src/playlist-maker.js

```diff
@@ -0,0 +1,57 @@
+var playItem = require('./playitem.js');
+var setupAutoadvance = require('./autoadvance.js');
+
+var playlistMaker = function(player, plist) {
```

Comment:
I think this can entirely be chalked up to different JS styles, but as the comment on the `player` var shows, it took me a full read through before I realized that this was basically an initializer.

A quick comment describing that `playlistMaker` is an initializer and returns an instance of `playlist` would have been really helpful on the first pass, particularly for those curmudgeons among us that still cling to the `new` keyword.